Moderation_Matrix.md

```diff
@@ -0,0 +1,50 @@
+# Moderation Matrix
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project (Rules 11 and 12)
+-Mild unprofessionalism: see this guide https://www.bitrix24.com/about/blogs/communications/instant-messaging-and-group-chat-etiquette-in-a-professional-environme.php 
+  - Drug related conversation (Rule 7)
+  - Special attention to treating everyone with respect
+  - Mental health issues discussion (Rule 12)
+- A second infraction of the following:
+  - Short circuiting a teaching moment (Rule 10)
+- Asking for help with Windows or encouraging the use of it
+  - “Chat bombing”, aka posting many short (ususally 4-5 or fewer words) messages in rapid succession, repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. (Use shift + enter and send ONE message where possible or edit your message) (Rule 7)
+
+## 2 Zaps (WARNING)
+Medium severity violations such as: 
+- Suggesting piracy (Rule 9)
+- Unsolicited pings/DM/friend request (Rule 6)
+- Political or religious topics (Rule 8)
+- Mild toxicity (directed insults, provoking/baiting other users, passive aggression, naming & shaming) (Rule 7)
+- Self promo without permission (ask with Modmail)(Rule 7)
+  - Do not share resources you have created solely for personal or monetary gain
```

Comment:
Can I get clarity on what is intended by resources created solely for personal gain?

Reply:
I wasn't a part of that part of the convo, but I took it as people who are blogging to build a platform, and wanting to share stuff here for a larger audience and there's zero relation to TOP or issues a learner might be encountering.

Reply:
@twalton83 @CouchofTomato 

The discussion was mostly about people that show up in the server simply to plug their stuff. It could be their programming server, they're own content, or anything.

It's to prevent people who just shamelessly self-promote from permeating our server. It's a form of spam and free advertisement.

Reply:
Right. That makes sense. Thanks.